ci/cscs-ci.yml

```diff
@@ -3,44 +3,67 @@ include:
 
 stages:
   - build
-  - test
+  - benchmark
 
 variables:
-  PERSIST_IMAGE_NAME: $CSCS_REGISTRY_PATH/cp2k:$CI_COMMIT_SHORT_SHA
+  PERSIST_IMAGE_NAME: ${CSCS_REGISTRY_PATH}/cp2k:${CI_COMMIT_SHORT_SHA}
+  PODMAN_TAR_FILE: /capstor/scratch/cscs/mkrack/cp2k_${CI_COMMIT_SHORT_SHA}.tar
 
 build cp2k eiger:
   extends: .container-builder-cscs-zen2
   stage: build
   timeout: 1h
-  before_script:
-    - echo "BUILD_IMAGE=${PERSIST_IMAGE_NAME}" > build.env
-    - 'echo "INFO: Building image ${PERSIST_IMAGE_NAME}"'
-  artifacts:
-    reports:
-      dotenv: build.env
+  after_script:
+    - echo PERSIST_IMAGE_NAME=${PERSIST_IMAGE_NAME}
+    - podman images
+    - podman save -o ${PODMAN_TAR_FILE} ${PERSIST_IMAGE_NAME}
```

Comment:
Sorry if my explanation was unclear, but this should only be required if you are testing/running things locally. In CI, the container image are pushed to an OCI registry, and `podman`/`sarus` can pull directly from it, without needing to go through a tar file for going from `podman` to `sarus`.